devel/nodejs12/Portfile

```diff
@@ -10,7 +10,7 @@ configure.cxx_stdlib    libc++
 PortGroup               cxx11 1.1
 
 name                    nodejs12
-version                 12.2.0
+version                 12.4.0
 revision                1
```

Comment:
Please reset the `revision` to 0, since `version` is being increased.

Reply:
Thanks, fixed!